domain/port/service/watcher.go

```diff
@@ -103,17 +105,17 @@ func (s *WatchableService) endpointToMachineMapper(
 		return e.Changed()
 	})
 
-	machineUUIDs, err := s.st.GetMachinesForEndpoints(ctx, endpointUUIDs)
+	machineNames, err := s.st.GetMachinesForEndpoints(ctx, endpointUUIDs)
```

Comment:
Rename to `GetMachineNamesByEndpoints`

Reply:
How about `GetMachineNamesForEndpoints`?